scriptspony/config/environment.py

```diff
@@ -12,7 +12,7 @@
 
 def load_environment(global_conf,app_conf):
     ## Hack to make our sqlalchemy config depend on scripts user
-    url = ('mysql://sql.mit.edu/%s+scripts-pony?read_default_file=~/.my.cnf'
+    url = ('mysql://sql.mit.edu/%s+scripts-pony?read_default_file=~/.sql/my.cnf'
```

Comment:
I'm not entirely sure why, but I believed that ~/.my.cnf didn't have the right information in my locker, then I accidentally committed the change. Will revert.